### PR TITLE
Get Query String Parameters

### PR DESCRIPTION
**Pull request by axgcoder**

### CODE REVIEW

**Comment by axgcoder** [comment]
Using this JS snippet, we can retrieve  query string parameters of url.

File: user snippets/getUrlParameter.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ * Run a small snippet of JavaScript during a mabl flow/journey
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+
+  let url_string = document.URL; // gets the current URL
+  let url = new URL(url_string);
+  let Id = url.searchParams.get("Id");// give the paramter to retrieve
```
Comment (srvance) [inline user snippets/getUrlParameter.js]: You could make this more general purpose with a variable, e.g., `queryParamName` and changing the last two lines to

```
let value = url.searchParams.get(mablInputs.variables.user.queryParamName);
callback(value);
```

**Review by srvance — COMMENTED** [review]
Thanks you for the contribution, @axgcoder! You're our first non-mabl contributor!

This is a really useful contribution. I just have a suggestion to make it a little more reusable for people.

**Comment by chougeenyang** [comment]
Thanks @axgcoder! We want to show our appreciation with some mabl swag :) You can use this link for a _free_ T-shirt, on us: https://mabl-swag.myshopify.com and use the Coupon Code: PTCKD1S1KD95

File: user snippets/getUrlParameter.js (hunk 1 above)
Comment (axgcoder) [reply user snippets/getUrlParameter.js]: Sure @srvance ,I will implement the changes and update it

**Comment by axgcoder** [comment]
> Thanks @axgcoder! We want to show our appreciation with some mabl swag :) You can use this link for a _free_ T-shirt, on us: https://mabl-swag.myshopify.com and use the Coupon Code: PTCKD1S1KD95

**Review by johnkbrew — APPROVED** [review]
Nice work, @axgcoder, thanks for the contribution!

**Review by srvance — APPROVED** [review]

**Pull request merged**